FRAuth/FRAuth/Callbacks/DeviceSigningVerifierCallback.swift

```diff
@@ -168,7 +168,7 @@ open class DeviceSigningVerifierCallback: MultipleValuesCallback, Binding {
                                authInterface: DeviceAuthenticator,
                                _ completion: @escaping DeviceSigningResultCallback) {
         
-        authInterface.initialize(userId: userKey.userId, prompt: Prompt(title: title, subtitle: subtitle, description: promptDescription))
+        let _ = authInterface.initialize(userId: userKey.userId, prompt: Prompt(title: title, subtitle: subtitle, description: promptDescription))
```

Comment:
Is there a reason why this `initialize` is not void? I seem to be missing something, so apologies in advance if this is obvious.
However, given that this returns it would sit better with me to use the "new" object in the method beyond that line.

Reply:
@george-bafaloukas-forgerock changed initialize to return Void.